src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx

```diff
@@ -31,12 +33,22 @@ const PanelSideBarItem = (props: PanelSideBarItemProps) => {
         style={{ paddingLeft: depth ? `${depth + 1}rem` : undefined }}
       >
         {hasitem ? (
-          <div>
-            <a role="button" className={classNames("nav-link", { "dropdown-toggle": hasitem })} onClick={() => setIsOpen(!isOpen)}>
-              {item.icon && <FontAwesomeIcon className="me-2" icon={item.icon} />}
-              <div className="text-justify">{item.title}</div>
-            </a>
-          </div>
+          <Row>
+            {/* Dropdown label/link */}
+            <Col lg={10} xs={10} onClick={() => !collapseIconOnly && setIsOpen(!isOpen)}>
+              <LinkRenderer item={item}>
```

Comment:
If it's not collapseIconOnly , you render a linkRendered as well?

Reply:
@manni497 Fixed